setup.py

```diff
@@ -51,7 +51,7 @@ def get_tests_requirements():
 header_count, long_description = get_description()
 
 setup(
-    name="Minigrid",
+    name="minigrid",
```

Comment:
Does setuptool just make everything lower case?

Reply:
`pip install -e` will install with this name, case sensitive – I'm not sure why this was working for `setuptools`

Reply:
If it is case sensitive, could you revert this change